plugin/xmark.vim

```diff
@@ -38,6 +38,7 @@ let s:files = {
 \ 'xsize':  s:dir . '/ext/xsize'
 \ }
 let s:app = 'Google Chrome'
+let s:app_loc = '/Applications/Google Chrome.app'
```

Comment:
I'm not sure, but hard-coded path makes me concerned. Is it possible on OS X to install Chrome on a different path than this? And if that's the case, and Chrome is not found there, we have no way to figure out the right path unless user provides, and we may have to fall back to `Google Chrome` as before.